polkadot/node/subsystem-util/src/lib.rs

```diff
@@ -399,21 +458,39 @@ impl Validator {
 
 		let validators = validators?;
 
-		Self::construct(&validators, signing_context, keystore)
+		// TODO: https://github.com/paritytech/polkadot-sdk/issues/1940
+		// When `DisabledValidators` is released remove this `if`` and add a
+		// `request_disabled_validators` call in the `try_join!` call above
+		let disabled_validators = if has_required_runtime(
+			sender,
+			parent,
+			RuntimeApiRequest::DISABLED_VALIDATORS_RUNTIME_REQUIREMENT,
+		)
+		.await
+		{
+			request_disabled_validators(parent, sender).await.await??
+		} else {
+			vec![]
+		};
+
+		Self::construct(&validators, &disabled_validators, signing_context, keystore)
```

Comment:
Ditto: This code needs to stay until we release v8: https://github.com/paritytech/polkadot-sdk/issues/1940